check.sh

```diff
@@ -151,7 +151,7 @@ function cmd_klippy() {
 }
 
 function cmd_test() {
-    run cargo test "${extraCargoArgs[@]}"
+    run cargo test "${extraCargoArgs[@]}" --features "itest/codegen-full"
```

Comment:
Full codegen appears to be enabled even though `--no-default-features` is set. Adding `itest/codegen-full` corrects the config of `itest`.

Reply:
This is on purpose -- as mentioned, local development is generally fine without the full codegen.

There is a tiny fraction of all changes that manifests only in the extended set of classes but not the limited one. The `godot-core` compile time is significantly higher if we do this by default, which is why it's disabled.

If you want, you could add a `--full` flag that enables both full + experimental codegen (similar to the existing `--double`). But it should be opt-in. Would also be good if it were its own commit.

Reply:
What I'm saying is that running `cargo test --no-default-features` still enables the `godot/default` feature, which enables `godot-codegen/codegen-full` but not `itest/codegen-full`. So far, I was unable to track down why `godot/default` gets enabled when it shouldn't.